README.md

```diff
@@ -172,6 +187,18 @@ const result = join(str, ' ')
 //    ^ 'hello world'
 ```
 
+### length
+
+This function is a strongly-typed counterpart of `String.prototype.length`.
+
+```ts
+import { length } from 'string-ts'
+
+const str = 'hello'
+const result = length(str)
+//    ^ 4
```

Comment:
4?

Reply:
Same with `St.Length<'hello'> // 4`

Reply:
Even though this PR implements a file called `math.ts` with complex TS arithmetics you can tell I suck at Math =D

Reply:
😂